### PR TITLE
Expose missing Quaternion operators.

### PR DESCRIPTION
**Pull request by 27thLiz**
Scripts can now evaluate the following cases:
- (quat \* real) and (quat / real)
- (quat + quat) and (quat - quat)

### CODE REVIEW

File: core/variant_op.cpp [hunk 1]
```diff
@@ -477,7 +477,13 @@ void Variant::evaluate(const Operator& p_op, const Variant& p_a, const Variant&
 					DEFAULT_OP_FAIL(MATRIX32);
 					DEFAULT_OP_LOCALMEM(+,VECTOR3,Vector3);
 					DEFAULT_OP_FAIL(PLANE);
-					DEFAULT_OP_FAIL(QUAT);
+					case QUAT: {
```
Comment (akien-mga) [inline core/variant_op.cpp]: Can't it be declared the same way as for `Vector3`, using `DEFAULT_OP_LOCALMEM(+,QUAD,Quad);`?

**Comment by Hinsbart** [comment]
> Can't it be declared the same way as for Vector3, using DEFAULT_OP_LOCALMEM(+,QUAT,Quat);?

Right, rebased.

**Comment by Hinsbart** [comment]
Mhh, CI failed on the android build, but it doesn't look like it was the code's fault ^^

**Comment by neikeq** [comment]
Restarted android build

File: core/variant_op.cpp [hunk 2]
```diff
@@ -699,7 +702,13 @@ void Variant::evaluate(const Operator& p_op, const Variant& p_a, const Variant&
 				DEFAULT_OP_FAIL(MATRIX32);
 				DEFAULT_OP_LOCALMEM_NUM(/,VECTOR3,Vector3);
 				DEFAULT_OP_FAIL(PLANE);
-				DEFAULT_OP_FAIL(QUAT);
+				case QUAT: {
```
Comment (akien-mga) [inline core/variant_op.cpp]: No `DEFAULT_OP_LOCALMEM_NUM(/, QUAT, Quat)` for this operation? (Just asking, I have no clue how this macro works :p)

Comment (Hinsbart) [inline core/variant_op.cpp]: `DEFAULT_OP_LOCALMEM_NUM` not only provides the operation on numbers but also on the type itself. 
However, Quat / Quat is not a valid op.

Comment (akien-mga) [inline core/variant_op.cpp]: Thanks for the clarification :)

**Pull request merged**